Kickstarter-iOS/Views/SettingsGroupedFooterView.swift

```diff
@@ -0,0 +1,36 @@
+import Prelude
+import UIKit
+
+private struct Padding {
+  static let horizontal: CGFloat = 8
+  static let vertical: CGFloat = 12
+}
+
+class SettingsGroupedFooterView: UITableViewHeaderFooterView {
```

Comment:
We had to customize the `UITableViewHeaderFooterView` because it would unfortunately work well with iOS 10. The auto layout on iOS 10 didn't work properly with this view, so we couldn't use the default `textLabel` or `detailTextLabel` but had to add a custom `UILabel` into the view hierarchy.